docs/README.md

```diff
@@ -468,6 +476,250 @@ Steps:
+The `istio.io/rev` label determines which control plane injects the sidecar proxy into the application pods.
+Additional namespace labels determine whether the control plane discovers and manages the resources in the namespace. 
+A control plane will discover and manage only those namespaces that match the discovery selectors configured on the control plane.
+Additionally, discovery selectors determine which control plane creates the `istio-ca-root-cert` ConfigMap in which namespace.
+
+Currently, discovery selectors in multiple control planes must be configured so that they don't overlap (i.e. the discovery selectors of two control planes don't match the same namespace).
+Each control plane must be deployed in a separate Kubernetes namespace.
+
+This guide explains how to set up two meshes: `mesh1` and `mesh2` in namespaces `istio-system1` and `istio-system2`, respectively, and three application namespaces: `app1`, `app2a`, and `app2b`.
+Mesh 1 will manage namespace `app1`, and Mesh 2 will manage namespaces `app2a` and `app2b`.
+Because each mesh will use its own root certificate authority and configured to use a peer authentication policy with the `STRICT` mTLS mode, the communication between the two meshes will not be allowed. 
+
+### Prerequisites
+
+- Install [istioctl](common/install-istioctl-tool.md).
+- Kubernetes 1.23 cluster.
+- kubeconfig file with a context for the Kubernetes cluster.
+- Install the Sail Operator and the Sail CRDs to the cluster.
+
+### Installation Steps
+
+#### Deploying the control planes
+
+1. Create the system namespace `istio-system1` and deploy the `mesh1` control plane in it.
+   ```sh
+   $ kubectl create namespace istio-system1
+   $ kubectl label ns istio-system1 mesh=mesh1
+   $ kubectl apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: mesh1
+   spec:
+     namespace: istio-system1
+     version: v1.24.0
+     values:
+       meshConfig:
+         discoverySelectors:
+         - matchLabels:
+             mesh: mesh1
+   EOF
+   ```
+   
+2. Create the system namespace `istio-system2` and deploy the `mesh2` control plane in it.
+   ```sh
+   $ kubectl create namespace istio-system2
+   $ kubectl label ns istio-system2 mesh=mesh2
+   $ kubectl apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: mesh2
+   spec:
+     namespace: istio-system2
+     version: v1.24.0
+     values:
+       meshConfig:
+         discoverySelectors:
+         - matchLabels:
+             mesh: mesh2
+   EOF
+   ```
+
+3. Create a peer authentication policy that only allows mTLS communication within each mesh.
+   ```sh
+   $ kubectl apply -f - <<EOF
+   apiVersion: security.istio.io/v1
+   kind: PeerAuthentication
+   metadata:
+     name: default
+     namespace: istio-system1
+   spec:
+     mtls:
+       mode: STRICT
+   EOF
+   
+   $ kubectl apply -f - <<EOF
+   apiVersion: security.istio.io/v1beta1
```

Comment:
I guess the apiVersion here should be `security.istio.io/v1` and not `security.istio.io/v1beta`.
The example in line 551 does not match line 562.

Reply:
Thanks. Fixed.